python/ccxt/bybit.py

```diff
@@ -541,6 +541,27 @@ def change_position_mode(self, is_hedge_mode, symbol=None, pair=None):
         except NotChanged:
             pass
 
+    def get_position_mode(self):
+        _type = self.safe_string(self.options, 'defaultType')
+        if _type == "linear":
+            self.load_markets()
+            response = self.privateGetPrivateLinearPositionList()
+            open_positions_mode_count = {"BothSide": 0, "MergedSingle": 0}
+            all_symbols_mode_count = {"BothSide": 0, "MergedSingle": 0}
+            positions = self.safe_value(response, 'result')
+            for _position in positions:
+                position = self.safe_value(_position, "data", _position)
+                if position:
+                    mode = self.safe_value(position, "mode")
+                    if self.safe_value(position, "size", 0):
+                        open_positions_mode_count[mode] += 1
+                    all_symbols_mode_count[mode] += 1
+            if open_positions_mode_count["BothSide"] == open_positions_mode_count["MergedSingle"]:
+                return all_symbols_mode_count["BothSide"] > all_symbols_mode_count["MergedSingle"]
```

Comment:
We still need to decide what to do when user has no open positions

Reply:
Let's make it simpler and take hedge mode if there's more than 3 symbols with bothside in that case.